win32ss/user/winsrv/consrv/frontends/gui/guiterm.c

```diff
@@ -229,13 +229,11 @@ GuiConsoleInputThread(PVOID Param)
                  * that the window that we want to destroy doesn't exist yet.
                  * So first empty the message queue.
                  */
-                /*
                 while (PeekMessageW(&TempMsg, NULL, 0, 0, PM_REMOVE))
                 {
                     TranslateMessage(&TempMsg);
```

Comment:
I don't know whether that TranslateMessage is necessary, though...

Reply:
It's probably not, I was just going for the smallest change/full revert. Would you prefer if I remove it?

Reply:
Yeah this is useless here, so you can remove it altogether.